Gems/ROS2/Code/Source/Sensor/SensorConfiguration.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/Sensor/SensorConfiguration.h"
```

Comment:
This include should be angle brackets.
```suggestion
#include <ROS2/Sensor/SensorConfiguration.h>
```